llvm/test/tools/llvm-cas-dump/basic_debug_test.ll

```diff
@@ -7,6 +7,9 @@
 
 ; REQUIRES: aarch64-registered-target
 
+; LLVM_ENABLE_ONDISK_CAS is not enabled by default on Windows
```

Comment:
Same here. We should create a `lit.local.cfg` here.

Reply:
Done